src/Microsoft.ML.Data/Transforms/ConcatEstimator.cs

```diff
@@ -3,33 +3,41 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.StaticPipe;
 using Microsoft.ML.StaticPipe.Runtime;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.Transforms
 {
-    public sealed class ConcatEstimator : IEstimator<ITransformer>
+    public sealed class ColumnConcatenatingEstimator  : IEstimator<ITransformer>
     {
         private readonly IHost _host;
         private readonly string _name;
         private readonly string[] _source;
 
-        public ConcatEstimator(IHostEnvironment env, string name, params string[] source)
+        /// <summary>
+        /// Initializes a new instance of <see cref="ColumnConcatenatingEstimator"/>
+        /// </summary>
+        /// <param name="env">The private instance of <see cref="IHostEnvironment"/>.</param>
```

Comment:
>instance [](start = 42, length = 8)

What does `private` mean in this context? I find it a little confusing calling this parameter `private`.